### DISCUSSION TITLE
SQL query question

### DISCUSSION THREAD

**Discussion by andreas-bertelsen** [aleksandervestlund/theatre-database#10] [Q&A] [answered]
How does the query for finding actors in the same act work?

**Comment by aleksandervestlund** [comment, marked as answer]
All actors are cross joiner with each other. The logic then filters out all actors not in any same act.

**Comment by chrishayesUMD** [comment]
how do you join table in sql??

**Reply by aleksandervestlund** [reply]
See this helpful link: <https://www.w3schools.com/sql/sql_join.asp>.
